java-frontend/src/test/files/metrics/Methods.java

```diff
@@ -36,4 +36,8 @@ int method(){  // counted
     }
   };
   abstract int method();  // counted
-}
\ No newline at end of file
+}
+
+record R(int x) {
+  int neg() { return -x; }
```

Comment:
We probably should add "//counted" comment next to the method for documentation consistency

Reply:
Good catch. Will do. Looks like I didn't look at the existing code carefully enough.